includes/controllers/class-attribute.php

```diff
@@ -183,7 +183,7 @@ function( $name ) {
 			$taxonomy = hp\prefix( $model_name . '_category' );
 
 			if ( taxonomy_exists( $taxonomy ) ) {
-				wp_set_post_terms( $post->ID, [ absint( $request->get_param( hp\prefix( 'categories' ) ) ) ], $taxonomy );
+				wp_set_post_terms( $post->ID, sanitize_key( $request->get_param( hp\prefix( 'categories' ) ) ), $taxonomy );
```

Comment:
`sanitize_key` accepts strings. Is value of `$request->get_param( hp\prefix( 'categories' )` is string even it is an array of values?